WEAPON/CreateWeaponObject.md

```diff
@@ -5,26 +5,21 @@ ns: WEAPON
 
 ```c
 // 0x9541D3CF0D398F36 0x62F5987F
-Object CREATE_WEAPON_OBJECT(Hash weaponHash, int ammoCount, float x, float y, float z, BOOL showWorldModel, float scale, Any p7);
+Object CREATE_WEAPON_OBJECT(Hash weaponHash, int ammoCount, float x, float y, float z, BOOL showWorldModel, float scale, cs_type(Any) BOOL scriptHostObject);
```

Comment:
last argument (p7) is `customModelHash` (integer). Remember that `Any` to `int` does not need `cs_type`.

Reply:
Should be good